cfgmgr/portmgr.cpp

```diff
@@ -10,6 +10,10 @@
 using namespace std;
 using namespace swss;
 
+/* Port default admin status is down */
+#define DEFAULT_ADMIN_STATUS_STR    "down"
```

Comment:
The default port admin status must be up (see also https://github.com/Azure/sonic-swss/issues/623)

Reply:
I think, the default must be down.

Reply:
Currently default is down anyway. IMO if not specified explicitly in config_db.json it should be 'down'.

Reply:
Currently default is 'down'. IMO if it is not specified explicitely in config_db.json it should be 'down'

Reply:
@stepanblyschak The default was up if it wasn't specified in the config_db.json and the port had an IP (as of July release). Recent change broke/changed this and also caused older configs to not work and be backwards incompatible. It is not acceptable for customers to move from one release to another and all of a sudden have their configs not working.

Reply:
@nikos-github i think this pull request doesn't change the current behavior. the admin status 'up' is set by converting the information from the minigraph to the configuration database, where the admin status is set to up for all ports that have IP addresses.